cmd/sonobuoy/app/images.go

```diff
@@ -22,46 +22,102 @@ import (
 
 	"github.com/heptio/sonobuoy/pkg/errlog"
 	"github.com/heptio/sonobuoy/pkg/image"
+	"github.com/heptio/sonobuoy/pkg/image/docker"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
-	"github.com/spf13/pflag"
 )
 
 var imagesflags imagesFlags
 
-type imagesFlags struct {
-	e2eRegistryConfig string
-	plugin            string
-	kubeconfig        Kubeconfig
-
-	imagesflags *pflag.FlagSet
-}
+const numDockerRetries = 1
+)
 
-func ImagesFlagSet(cfg *imagesFlags) *pflag.FlagSet {
-	flagset := pflag.NewFlagSet("images", pflag.ExitOnError)
-	AddPluginFlag(&cfg.plugin, flagset)
-	AddKubeconfigFlag(&cfg.kubeconfig, flagset)
-	cfg.imagesflags = flagset
-	return flagset
+type imagesFlags struct {
+	e2eRegistryConfig  string
+	plugin             string
+	username           string
+	password           string
+	imagesSaveFileName string
```

Comment:
This isnt used either; I'm guessing you thought about having it as a flag and cut it out?

I do think that would be a reasonable thing to do so they dont have to find what file it was after the fact (same small UX issue that retrieve used to have and we just decided to echo it out to stdout so it could be grabbed)